customers.sql

```diff
@@ -0,0 +1,11 @@
+select 
+  c.id as customer_id, 
+  c.name, 
+  c.email, 
+  min(o.created_at) as first_order_at, 
```

Comment:
Comment here! This code must be changed in some way.

Reply:
Done! See latest SQL for changes